pkg/resource/files.go

```diff
@@ -123,13 +123,13 @@ func (r *Files) Walk(fn WalkFunc) error {
 		}
 
 		if info.Mode()&os.ModeSymlink != 0 {
-			logger.Info("skipping symlink: path=%q", relPath)
+			r.Info(logger.ScanError, "skipping symlink: path=%q", relPath)
```

Comment:
```suggestion
			r.Info(logger.ScanDetail, "skipping symlink: path=%q", relPath)
```

Rationale: See `logger.CloneDetail` comment